unity-renderer/Assets/Rendering/Outliner/OutlineScreenEffectFeature.cs

```diff
@@ -46,25 +49,39 @@ public void Setup(RenderTargetIdentifier source, RenderTargetHandle destination,
 
         public override void Execute(ScriptableRenderContext context, ref RenderingData renderingData)
         {
-            material.SetColor(_InnerColor, settings.innerColor);
-            material.SetColor(_OutlineColor, settings.outlineColor);
-            material.SetFloat(_OutlineSize, settings.outlineSize);
-            material.SetColor(_BlurColor, settings.blurColor);
-            material.SetFloat(_BlurSize, settings.blurSize);
-            material.SetFloat(_Sigma, settings.blurSigma);
-            material.SetFloat(_Fade, settings.effectFade);
+            if (!IsOutlineAvailable()) return;
+
+            material.SetColor(INNER_COLOR, settings.innerColor);
+            material.SetColor(OUTLINE_COLOR, settings.outlineColor);
+            material.SetFloat(OUTLINE_SIZE, settings.outlineThickness.Get());
+            material.SetColor(BLUR_COLOR, settings.blurColor);
+            material.SetFloat(BLUR_SIZE, settings.blurSize.Get());
+            material.SetFloat(SIGMA, settings.blurSigma.Get());
+            material.SetFloat(FADE, settings.effectFade);
             CommandBuffer cmd = CommandBufferPool.Get(PROFILER_TAG);
 
             using (new ProfilingScope(cmd, new ProfilingSampler(PROFILER_TAG)))
```

Comment:
why you commented `ProfilingScope` in `OutlineMaskFeature` but left it here?

Reply:
The other one throws a millon errors per frame when deep profiling and I couldn't fix it :(